ee/clickhouse/models/action.py

```diff
@@ -116,16 +116,22 @@ def filter_element(filters: Dict, prepend: str = "") -> Tuple[List[str], Dict]:
         params["{}tag_name_regex".format(prepend)] = r"(^|;){}(\.|$|;|:)".format(filters["tag_name"])
         conditions.append("match(elements_chain, %({}tag_name_regex)s)".format(prepend))
```

Comment:
WDYT about move this function together with the rest of the filters logic files over at ee/clickhouse/models/property.py

Reply:
Done! I reflexively tried to go to property tests too to find the tests for this function

Reply:
It does however cause a circular import so we'll have to import within a function to resolve it

Reply:
I think that's fine :)